packages/java-edition/src/mcfunction/parser/argument.ts

```diff
@@ -87,6 +87,7 @@ function shouldValidateLength(ctx: core.ParserContext) {
  */
 export const argument: mcf.ArgumentParserGetter = (
 	rawTreeNode,
+	ctx: core.ContextBase
 ): core.Parser | undefined => {
```

Comment:
It makes sense to pass the `ctx` to the completer `getMockNodes`, but this function is already returning a `Parser`, which is a function that has access to `ctx`.